cicecore/cicedynB/infrastructure/ice_grid.F90

```diff
@@ -2407,7 +2401,7 @@ subroutine get_bathymetry
          do iblk = 1, nblocks
             do j = 1, ny_block
             do i = 1, nx_block
-               k = kmt(i,j,iblk)
+               k = min(nint(kmt(i,j,iblk)),nlevel)
```

Comment:
Is this also something that was found with signalling NaNs ? not sure what this is changing ...